memory-viz/src/memory_model.ts

```diff
@@ -743,13 +743,23 @@ export class MemoryModel {
                 this.item_min_height
             );
 
-            this.drawText(
-                attribute,
-                x + this.item_min_width / 2,
-                curr_y + this.item_min_height / 2 + this.font_size / 4,
-                style.text_value,
-                stack_frame ? "variable" : "attribute"
-            );
+            if (attribute.trim() === "") {
```

Comment:
This check is good, but I don't want to actually draw a blank box for the attribute name, but instead just draw nothing at all. (In other words, you can have a guard for `attribute.trim() !== ""` to prevent `drawText` from being called unnecessarily, but don't include the box.)

I guess this makes the change seem a bit less significant since originally I'd asked about `null` keys, which we figured out can't actually be entered with JSON. However, it's still good to prevent redundant text at all from being drawn!